include/umf/providers/provider_os_memory.h

```diff
@@ -61,6 +71,8 @@ typedef enum umf_numa_mode_t {
 typedef struct umf_os_memory_provider_params_t {
     /// Combination of 'umf_mem_protection_flags_t' flags
     unsigned protection;
+    /// mmap flag (Linux only, ignored on Windows)
```

Comment:
Why is this ignored on windows? Don't we want to have the same interface on linux and windows for shared memory eventually?

Reply:
@ldorau we had to mention this in our documentation

Reply:
@ldorau @igchor Also I think we could return NOT_SUPPORTED for windows

Reply:
Done. It will return `UMF_RESULT_ERROR_INVALID_ARGUMENT` for Windows

Reply:
Why not NOT_SUPPORTED as @bratpiorka suggested? I think it fits better for cases where something is only supported on one platform.

Reply:
OK, I will do that. It requires the following changes - please review: https://github.com/oneapi-src/unified-memory-framework/pull/438 and https://github.com/oneapi-src/unified-memory-framework/pull/439

Reply:
Done

Reply:
@igchor Done